server/aws-lsp-codewhisperer/src/language-server/inline-completion/codeWhispererServer.ts

```diff
@@ -483,18 +484,24 @@ export const CodewhispererServerFactory =
                                 supplementalContextFromEdits =
                                     await recentEditTracker.generateEditBasedContext(textDocument)
                             }
-                            const editPredictionAutoTriggerResult = editPredictionAutoTrigger({
-                                fileContext: fileContext,
-                                lineNum: params.position.line,
-                                char: triggerCharacter,
-                                previousDecision: previousDecision,
-                                cursorHistory: cursorTracker,
-                                recentEdits: recentEditTracker,
-                            })
-                            predictionTypes = [
-                                ...(autoTriggerResult.shouldTrigger ? [['COMPLETIONS']] : []),
-                                ...(editPredictionAutoTriggerResult.shouldTrigger && editsEnabled ? [['EDITS']] : []),
-                            ]
+                            if (previousDecision === 'Accept' && previousSuggestionType === SuggestionType.EDIT) {
```

Comment:
question: Do you need to check editsEnabled here?

Reply:
> Do you need to check editsEnabled

should be fine cuz it's checked at L481

Reply:
as a short term hack, could we add another field called prevoiusDecisionType `previousDecisionTimestamp` and only go this path if timestamp is less than a threshold, for example 1000ms or 2000ms

```
if (previousDecision === 'Accept' && previousSuggestionType === 'Edits' && clock.now() - previousDecisionTimestamp <= threshold) {
        predictionTypes = [['EDITS']]
} else {
      // normal flow
}
```

Reply:
long term solution we maybe still want to have something like sticky session which requires introducing another new field in the completion API. But let's start with this quick short term for now to unblock us.